test/Engine/FFITarget/TestObjects.cs

```diff
@@ -457,4 +457,35 @@ public void Dispose()
         }
     }
 
+    public class TestRankReduce : IDisposable
+    {
+        private string RankReduceTestField;
+
+        [AllowRankReduction]
+        public List<string> RankReduceArrowProperty => new List<string> { RankReduceTestField };
```

Comment:
How is this syntax different from the `RankReducePropertyGetter`?

Reply:
It's just syntactic sugar for the getter. Testing one of these would suffice.

Reply:
Fair enough. I just carried this over from the test cases I used during development. I removed the arrow property and added a method test (not sure if this exists anywhere else).